MsfinderCommon/Utility/FileStorageUtility.cs

```diff
@@ -473,20 +473,48 @@ public sealed class FileStorageUtility {
                 return folderPath + "/" + formula + "." + SaveFileFormat.sfd;
             }
         }
+        public static string GetSmileLogFilePath(string folderPath)
+        {
+            folderPath = Path.GetDirectoryName(folderPath);
 
-        public static void DeleteSfdFiles(string[] structureFiles) {
-            foreach (var file in structureFiles) {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                return folderPath + "\\" + "log_smiles.smi";
+            }
+            else
+            {
+                return folderPath + "/" + "log_smiles.smi";
+            }

```

Comment:
Isn't there some built-in function which can handle paths in a platform independent way?

Reply:
updated in b37f876